apps/cookbook/src/app/examples/charts-example/examples/stock-config/chart-example-config-basic-stock.component.ts

```diff
@@ -0,0 +1,79 @@
+import { AfterViewInit, Component, OnDestroy } from '@angular/core';
```

Comment:
Consider renaming this file and everything else that is named basic to base instead